Automattic/wp-calypso
# Theme Showcase: Do not show wporg themes if they are also retired wpcom themes

Related to #78091

## Proposed Changes

The current implementation shows a wporg theme even if its wpcom counterpart is retired. For instance, the theme Maxwell exists in both wporg and wpcom, but it's retired in wpcom. This leads to a scenario where the Theme Showcase will show the wporg version of the theme, but when the user clicks on the theme, the detail page will say that the theme is retired. This behavior is confusing, and might lead to issues as described in #78091.

This PR updates the logic so that retired wpcom themes are always omitted from the Theme Showcase, even if their wporg counterparts exist. In other words, when users search for Maxwell, even though the theme exists in wporg, it will not be shown in the Theme Showcase listing because it's retired in wpcom.

## Testing Instructions



* Head to the Theme Showcase.
* Search for a retired theme that also exists in wporg, such as Maxwell or Adelle.
* Ensure that the theme is not shown in the theme listing.

## Pre-merge Checklist
